foundation_cms/static/scss/redesign_migrated_content.scss

```diff
@@ -4,16 +4,34 @@
 // Our custom variables and base styles
 @import "../scss/colors";
 @import "../scss/type";
+@import "../scss/buttons";
 
 // Site-wide Components
 @import "../scss/components/primary_nav";
 @import "../scss/components/footer";
 @import "../scss/components/newsletter_signup/newsletter_signup_form";
 
-/* Base font styles */
+/* Overriding legacy SCSS that was targeting these elements */
 .site-footer-ns,
-.primary-nav {
-  font-family: $body-font-family;
-  font-size: 1rem;
-  line-height: 1.5;
+.primary-nav-ns {
+  ul,
+  ol,
+  p {
+    font-family: $body-font-family;
+    font-size: unset;
+    line-height: 1.3;
+  }
+
+  a.btn-primary {
+    &:hover {
+      background-color: color(neutral, "600");
+      border-color: black;
+    }
+
+    &.dark-mode {
+      &:hover {
+        background-color: color(orange, "100");
+      }
+    }
+  }
 }
```

Comment:
This is the latest update I made. I am pretty much just defining the hover state for the buttons and updating some font stuff for the footer text (it appears that bootstrap was targeting  `ul`, `ol`, and `p` elements specifically, as well as `<a>` tags with the class `btn-primary`).  

Note: While we technically have the rule `color(neutral, "600")` at our disposal, it currently does not work with Black or White, as they are not defined with lists of shades.

While I could update that in this PR, I have opted not too in the question of time, and to avoid over-complicating this PR.